client/packages/programs/src/api/hooks/useProgramList.ts

```diff
@@ -7,7 +7,7 @@ import { ProgramFragment } from '../operations.generated';
 import { useProgramsGraphQL } from '../useProgramsGraphQL';
 import { LIST, PROGRAM } from './keys';
 
-export const useProgramList = () => {
+export const useProgramList = (includeImmunisation?: boolean) => {
```

Comment:
I just made an additional configuration rather than creating a whole new hook for this. I assume we want *all* programs, including immunisation ones?